src/liveEditor/utils/liveEditorPostMessage.ts

```diff
@@ -6,8 +6,7 @@ let liveEditorPostMessage: EventManager | undefined;
 if (typeof window !== "undefined") {
     liveEditorPostMessage = new EventManager(LIVE_EDITOR_CHANNEL_ID, {
         target: window.parent,
-        debug: false,
-        suppressErrors: true,
+        debug: true,
     });
 }
 
```

Comment:
Let's restore this file. If you don't see the suppressErrors option, update the advanced post message package